ext/exception_serialize.c

```diff
@@ -36,27 +38,21 @@ static zend_result dd_exception_to_error_msg(zend_object *exception, void *conte
         default: exception_type = "Thrown"; break;
     }
 
-    int error_len = asprintf(&error_text, "%s %s%s%s%s in %s:" ZEND_LONG_FMT, exception_type,
+    asprintf(&error_text, "%s %s%s%s%s in %s:" ZEND_LONG_FMT, exception_type,
             ZSTR_VAL(exception->ce->name), status_line ? status_line : "", ZSTR_LEN(msg) > 0 ? ": " : "",
             ZSTR_VAL(msg), ZSTR_VAL(file), line);
 
     free(status_line);
 
-    ddtrace_string key = DDTRACE_STRING_LITERAL("error.message");
-    ddtrace_string value = {error_text, error_len};
-    zend_result result = add_tag(context, key, value);
+    ddog_add_span_meta(span, dd_str_to_CharSlice("error.message"), dd_str_to_CharSlice(error_text));
```

Comment:
```suggestion
    ddog_add_span_meta(span, DDOG_CHARSLICE_C("error.message"), (ddog_CharSlice) {.ptr = error_text, .len = error_len});
```
We don't have to recompute the len.